models/clopath_synapse.h

```diff
@@ -247,16 +247,17 @@ clopath_synapse< targetidentifierT >::send( Event& e, size_t t, const CommonSyna
   std::deque< histentry_extended >::iterator start;
   std::deque< histentry_extended >::iterator finish;
 
+  // facilitation due to postsynaptic activity since last pre-synaptic spike
```

Comment:
maybe move this comment down to the facilitate call to match the depress() one?

Reply:
Done!